tokio-console/src/main.rs

```diff
@@ -68,7 +71,15 @@ async fn main() -> color_eyre::Result<()> {
             warnings::Linter::new(warnings::NeverYielded::default()),
         ])
         .with_retain_for(retain_for);
-    let mut input = input::EventStream::new();
+    let mut input = Box::pin(input::EventStream::new().try_filter(|event| {
+        ready(!matches!(
```

Comment:
style nit, take it or leave it: my preference would have been to call this as
```suggestion
        future::ready(!matches!(
```
since there are a few different things named `ready` (eg the `ready!` macro) and it makes it a little clearer that we're constructing a future. not a big deal though!